components/MechPage/WhyOlasMech.jsx

```diff
@@ -0,0 +1,38 @@
+import { SECTION_BOX_CLASS, SUB_HEADER_LG_CLASS } from 'common-util/classes';
+import { InfoCardList } from 'components/InfoCardList';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { BicepsFlexed, HandCoins, Sparkles } from 'lucide-react';
+
+const list = [
+  {
+    title: 'Specialisation',
```

Comment:
currently correct from UK standpoint 😄 though Americans will 🤮 
```suggestion
    title: 'Specialization',
```